src/bridge.ts

```diff
@@ -65,13 +66,21 @@ class DenoBridge {
     // a malformed semver range. If this does happen, let's throw an error so
     // that the tests catch it.
     if (downloadedVersion === undefined) {
-      throw new Error('Could not read downloaded binary')
+      const error = new Error(
+        "There was a problem setting up the Edge Functions environment and it's unfortunately not possible to run Edge Functions from this platform. More on supported platforms here: https://deno.land/manual/getting_started/installation.",
```

Comment:
What do you think about a slightly different spin on the message? My thinking is:

- We should try to point people into self-serving a solution. If they're able to install Deno manually, the problem should go away
- It's possible that the problem comes from an unsupported platform, but we don't know for sure

```suggestion
        "There was a problem setting up the Edge Functions environment. To try a manual installation, visit https://deno.land/manual/getting_started/installation.",
```

We should also try to replace this with a `ntl.fyi` shortlink. I think @minivan is looking into getting us access. If that doesn't happen soon, we can merge as is but we should do a follow up.

Reply:
That's great! I've updated my suggestion accordingly.